api/src/tests/multisig_transactions_test.rs

```diff
@@ -411,13 +411,17 @@ async fn test_multisig_transaction_simulation() {
         .create_multisig_account(
             owner_account_1,
             vec![owner_account_2.address(), owner_account_3.address()],
-            2,    /* 2-of-3 */
```

Comment:
Are we testing the 2 of 3 case anywhere now?

Reply:
Although we have some 2-of-3 cases, I added some more (both positive and negative cases).